hub_map.html

```diff
@@ -316,22 +316,25 @@ <h2>Sunrise Hubs</h2>
 
 function makeGeoJSONFeature(coordinatorLines) {
   var line = coordinatorLines[0];
-  var state = line[0];
-  var city = line[1];
+  var state = line[0].trim();
+  var city = line[1].trim();
   var lat = parseFloat(line[2].replace(/"/, ""));
   var lng = parseFloat(line[3].replace(/"/, ""));
-  var facebookGroupLink = line[9];
-  var websiteEmail = line[10];
-  
+  var facebookGroupLink = line[9].trim();
+  var websiteEmail = line[10].trim();
+  var websiteLink = line[11].trim();
+  var twitterLink = line[12].trim();
+  var instagramLink = line[13].trim();
```

Comment:
As a side-note, I am strongly considering refactoring this code to use column headers for indexing (rather than raw numbers)... don't want someone to innocently reorder columns in the CSV and cause the hub map to break...